setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='ArchivesSpace API Python Scripts',
+    version='1.0.0',
+    description='',
+    packages=find_packages(exclude=['tests']),
+    author='Eric Hanson',
+    author_email='ehanson@mit.edu',
+    install_requires=[
```

Comment:
This list of packages needs to include everything that's listed in the Pipfile under the `[packages]` section. In looking at the Pipfile though I noticed that it's missing `attrs`. It's working because asnake is pulling in `attrs` but we should definitely have it listed in our Pipfile, too, since it's a direct third party dependency.